### ISSUE TITLE
Send Input Key

### ISSUE THREAD

**Issue by ajtruckle** [MicrosoftEdge/WebView2Feedback#4505]
### Describe the feature/enhancement you need

The WebView2 control is hosted on a CDialog. 

Whilst the control can detect hot keys and let us redirect them to the parent, the reverse is not possible. 

### The scenario/use case where you would use this feature

If my dialog has focus then it eats up the webview2 hotkey. I have to click into the control and the use the hotkey. 

### How important is this request to you?

Impactful. My app's user experience would be significantly compromised without it.

### Suggested implementation

I tried programmatically setting focus to the control and using SetInput. Does nothing. 

Can we please have a simple SendKeyValue that relays to webview2 to perform a default hotkey action.

**Comment by ajtruckle** [comment]
I hope that this can be implemented.

**Comment by victorhuangwq** [comment]
@bradp0721 I don't think this is currently possible, to send a key value from WebView2 control back to the parent. Let me know your thoughts? 

If it's not possible, we can track it and triage it accordingly

**Comment by ajtruckle** [comment]
@victorhuangwq 
@bradp0721 
I am talking about the reverse.
WebView2 has the accelerator key event. So you can track hotkeys in webview2 and reroute to parent. 

I am talking about giving parent ability to push a hotkey to webview2.

**Comment by ajtruckle** [comment]
@victorhuangwq @bradp0721 
Do you now understand my feature request correctly?

**Comment by ajtruckle** [comment]
@champnic 
It is still not clear to me that the WebView2 team understand my request. I have been patient with this fragmented conversation. 😊

**Comment by champnic** [comment]
Hey @ajtruckle - Sorry, yes I think this is the follow-up item to a discussion in a separate thread, where you are trying to inject the keyboard input using SendInput to trigger shortcut handling within the WebView2 to launch a feature. I forget which feature exactly that was. It looks like you are able to use SendInput for other scenarios though, or does that only work when the WebView2 already has focus? If you remember which issue that was please link it.

**Comment by ajtruckle** [comment]
Hi @champnic !
https://github.com/MicrosoftEdge/WebView2Feedback/issues/4483

**Comment by champnic** [comment]
Thanks! Memory is now paged back in :)

We already have an item tracking #3671 - Is this feature request intended to cover something that's not part of that item?

**Comment by ajtruckle** [comment]
@champnic  This is not an issue for me now (if seems)  because I can programmatically invoke the Find UI now when I handle CTRL + F. But the principle of the issue still remains unsolved. Just not a worry for me now.
